src/sympc/api.py

```diff
@@ -117,4 +118,6 @@
     ),
     ("sympc.tensor.ShareTensor.unsqueeze", "sympc.tensor.ShareTensor"),
     ("sympc.tensor.ShareTensor.view", "sympc.tensor.ShareTensor"),
+    # TODO: correct way to define a static function?
+    ("sympc.tensor.static.helper_argmax_pairwise", "sympc.tensor.ShareTensor"),
```

Comment:
You need to add:
`"sympc.tensor.static", sympc.tensor.static`
in the `allowed_external_modules`  (another list above this one)

Reply:
Thanks for the answer! Unfortunately, I get the same error when doing this. @gmuraru